pyproject.toml

```
@@ -23,13 +23,13 @@ classifiers = [
   "Topic :: Scientific/Engineering",
 ]
 dependencies = [
-  "numpy >=1.19",  # NEP29 + 1yr
-  "scipy >=1.5",   # Scientific Python SPEC 0 + 1yr
-  "nibabel >=3.0",
-  "pandas >=0.25.2",  # First with Py38 wheel
+  "numpy >=1.22",  # SPEC 0 + 1yr (3yr)
+  "scipy >=1.8",   # SPEC 0 + 1yr (3yr)
+  "nibabel >=4.0",  # SPEC 0 + 1yr (3yr)
+  "pandas >=1.4.0",  # SPEC 0 + 1yr (3yr)
   "formulaic >=0.2.4, <0.6",  # Tested on 0.2.4-0.5.2
   "sqlalchemy >=1.3.16",
-  "bids-validator>=1.11",  # Keep up-to-date to ensure support for recent modalities
+  "bids-validator>=1.14.12",  # Keep up-to-date to ensure support for recent modalities
```

Woops. Looked at the JS validator repo.

```suggestion
  "bids-validator>=1.14.7",  # Keep up-to-date to ensure support for recent modalities
```